src/hash.cr

```diff
@@ -643,6 +704,13 @@ class Hash(K, V)
     to_s(io)
   end
 
+  # Converts to a `String`, preserving all symbols including {}, :, and =>. 
```

Comment:
I think that's slightly misleading, since `{"foo": "bar"}.to_s # =>{"foo" => "bar"} "